regression/scripts/Vehicles/AS2.py

```diff
@@ -0,0 +1,448 @@
+    wing.chords.mean_aerodynamic = 3.3
+
+    wing.areas.reference         = 24.5    #
+    
+    wing.twists.root             = 3.0 * Units.degrees
+    wing.twists.tip              = 3.0 * Units.degrees  
+    
+    wing.origin                  = [46,0,0]
+    wing.aerodynamic_center      = [2,0,0]
+    
+    wing.vertical                = False 
+    wing.symmetric               = True
+    wing.high_mach               = True
+    wing.transition_x_upper      = 0.9
+    wing.transition_x_lower      = 0.9    
+    
+    wing.dynamic_pressure_ratio  = 0.9  
+    
+    # add to vehicle
+    vehicle.append_component(wing)
+    
+    
+    # ------------------------------------------------------------------
+    #   Vertical Stabilizer
+    # ------------------------------------------------------------------
+    
+    wing = SUAVE.Components.Wings.Wing()
+    wing.tag = 'vertical_stabilizer'    
+    
+    wing.aspect_ratio            = 1.3      #
+    wing.sweeps.quarter_chord    = 45 * Units.deg
+    wing.thickness_to_chord      = 0.08
+    wing.taper                   = 0.5
+    wing.span_efficiency         = 0.9
+    
+    wing.spans.projected         = 3.5      #    
+
+    wing.chords.root             = 6.0
+    wing.chords.tip              = 3.0
+    wing.chords.mean_aerodynamic = 4.0
+    
+    wing.areas.reference         = 33.9    #
+    
+    wing.twists.root             = 0.0 * Units.degrees
+    wing.twists.tip              = 0.0 * Units.degrees  
+    
+    wing.origin                  = [40,0,0]
+    wing.aerodynamic_center      = [2,0,0]    
+    
+    wing.vertical                = True 
+    wing.symmetric               = False
+    wing.t_tail                  = False
+    wing.high_mach               = True
+    wing.transition_x_upper      = 0.9
+    wing.transition_x_lower      = 0.9      
+    
+    wing.dynamic_pressure_ratio  = 1.0
+        
+    # add to vehicle
+    vehicle.append_component(wing)
+
+
+    # ------------------------------------------------------------------
+    #  Fuselage
+    # ------------------------------------------------------------------
+    
+    fuselage = SUAVE.Components.Fuselages.Fuselage()
+    fuselage.tag = 'fuselage'
+    
+    #fuselage.number_coach_seats    = vehicle.passengers
+    fuselage.seats_abreast         = 2
+    fuselage.seat_pitch            = 1
+    
+    fuselage.fineness.nose         = 4.0 # These finenesses are smaller than the real value due to limitations of existing functions
+    fuselage.fineness.tail         = 4.0
+    
+    fuselage.lengths.nose          = 12.
+    fuselage.lengths.tail          = 25.5
+    fuselage.lengths.cabin         = 11.5
+    fuselage.lengths.total         = 49.    
+    fuselage.lengths.fore_space    = 16.3
+    fuselage.lengths.aft_space     = 16.3  
+    
+    fuselage.width                 = 2.35
+    
+    fuselage.heights.maximum       = 2.55    #
+    fuselage.heights.at_quarter_length          = 4. # Not correct
```

Comment:
This script wasn't actually being used and the test case doesn't run properly. I'd say delete it for now.